pkgs/os-specific/linux/zfs/unstable.nix

```diff
@@ -11,17 +11,11 @@ let
 in
 callPackage ./generic.nix args {
   # check the release notes for compatible kernels
-  # NOTE:
-  #   zfs-2.1.9<=x<=2.1.10 is broken with aarch64-linux-6.2
-  #   for future releases, please delete this condition.
   kernelCompatible = if stdenv'.isx86_64 || removeLinuxDRM
     then kernel.kernelOlder "6.4"
     else kernel.kernelOlder "6.2";
 
-  latestCompatibleLinuxPackages = if stdenv'.isx86_64 || removeLinuxDRM then
-    linuxKernel.packages.linux_6_3
-  else
-    linuxKernel.packages.linux_6_1;
```

Comment:
If you can give me sometime, I will just look if ZFS works on 6_4.

Reply:
It works, please keep the if and just put 6_4 instead of 6_3.

Reply:
Sure, changing to draft.

Reply:
Also, no rush, I don't think dropping this kernel is time sensitive. :3

Reply:
2.1.12 and also the 2.2.0rc2 are currently just mentioning 6.3 as highest supported, so merging that as it is would mean to downgrade zfs users if the are using `latestCompatibleLinuxPackages` as kernel package.

Reply:
will wait for 2.2.0 release

Reply:
@Shawn8901 2.1.12 has already 6.4 support, 2.2 has 6.5 support

Reply:
Ah… Indeed the META have 6.3 maximal… Surprising.

Reply:
Let's proceed now.